packages/itwinui-css/src/toast/toast.scss

```diff
@@ -52,13 +52,13 @@
   z-index: 99999;
 
   @media screen and (max-width: 400px) {
-    width: calc(100vw - 36px);
-    margin-top: var(--iui-size-s);
+    inline-size: calc(100vw - 36px);
+    margin-block-start: var(--iui-size-s);
   }
 
   @media screen and (min-width: 401px) {
```

Comment:
Should this check for `min-width` be changed to `min-inline-size`?

Reply:
done

Reply:
no, logical properties do not work inside media queries.

Reply:
reverted